src/components/custom/_signup-affiliate-details.js

```diff
@@ -0,0 +1,398 @@
+                        <form style={{ display: 'block' }}>
+                            <Header as="h3" type="heading-3" mb="8px">
+                                {localize('We’re glad you’re here')}
+                            </Header>
+                            <Header as="p" type="subtitle-2" weight="normal">
+                                {localize(
+                                    'Complete this form to sign up for our partnership programme.',
+                                )}
+                            </Header>
+                            <InputGroup>
+                                {form_inputs.map((item) => {
+                                    return item.name === 'country' ? (
+                                        <DropdownSearchWrapper key={item.id}>
+                                            <DropdownSearch
+                                                id={item.id}
+                                                is_affiliate
+                                                selected_item={values.country}
+                                                default_item={''}
+                                                error={item.touch && item.error}
+                                                items={item.list}
+                                                label={localize('Country of residence')}
+                                                onChange={(value) =>
+                                                    setFieldValue('country', value)
+                                                }
+                                                onBlur={handleBlur}
+                                            />
+                                        </DropdownSearchWrapper>
+                                    ) : (
+                                        <Field
+                                            name={item.name}
+                                            key={item.id}
+                                            onChange={(value) => {
+                                                setFieldValue(item.name, value)
+                                            }}
+                                        >
+                                            {({ field }) => (
+                                                <Input
+                                                    {...field}
+                                                    id={item.id}
+                                                    is_date={item.name === 'date'}
+                                                    is_affiliate
+                                                    type={item.type}
+                                                    border="solid 1px var(--color-grey-7)"
+                                                    labelColor="grey-5"
+                                                    labelHoverColor="grey-5"
+                                                    labelFocusColor="black"
```

Comment:
Could you recheck the props on `Input`. Some of them have been globally refactored to be snake_case instead of camelCase now.

Reply:
Those props becomes unnecessary, removed.